src/map/tiles.c

```diff
@@ -20,6 +20,8 @@
 #include "map/terrain.h"
 #include "scenario/map.h"
 
+#include <stdio.h>
```

Comment:
Where is this include needed?

Reply:
This include was being used to do great things once. Actually ok it was being used to do pretty terrible things and I forgot to remove it.